regression-tests/test-results/pure2-deducing-pointers.cpp2.output

```diff
@@ -0,0 +1,12 @@
+pure2-deducing-pointers.cpp2...
+pure2-deducing-pointers.cpp2(14,8): error: = - pointer assignment from null or integer is illegal
```

Comment:
Thanks for including a new regression test.

Minor naming thing that I can clean up post-merge: Tests that are intended to fail cppfront compilation (not generate a .cpp file) should have a name that ends in `...error.cpp2`. That way my super-simple `run-tests.bat` test runner will count it as not intending to to create a `.cpp` and not flag a mismatch error. I'll take care of this post-merge though.

Reply:
Right... will correct that.

Reply:
Oh, I see that you have merged that. Should I prepare another PR with the name change?

Reply:
By the way. Thank you for merging that. Appreciate it!

Reply:
Yup, I did in another commit. All's well, thanks!